packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -63,6 +63,21 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     throw new NoDataError(`Cannot update ${this.collectionName}`);
   }
 
+  public async softDelete(data: Type): Promise<Type> {
```

Comment:
Hi

Can you create separate PR with soft delete change. 
This change is incomplete we will not be able to merge it even if others will work

Reply:
oki